src/bo4e/bo/ausschreibung.py

```diff
@@ -52,16 +52,14 @@ class Ausschreibung(Geschaeftsobjekt):
     Mit diesem Objekt können Geschäftspartner übertragen werden.
     Sowohl Unternehmen, als auch Privatpersonen können Geschäftspartner sein
     """
-    abgabefrist: Optional["Zeitraum"] = None
-    """
-    Diese Komponente wird zur Abbildung von Zeiträumen in Form von Dauern oder der Angabe von Start und Ende verwendet.
-    Es muss daher entweder eine Dauer oder ein Zeitraum in Form von Start und Ende angegeben sein
-    """
-    bindefrist: Optional["Zeitraum"] = None
-    """
-    Diese Komponente wird zur Abbildung von Zeiträumen in Form von Dauern oder der Angabe von Start und Ende verwendet.
-    Es muss daher entweder eine Dauer oder ein Zeitraum in Form von Start und Ende angegeben sein
-    """
+    abgabefrist: Optional["Zeitspanne"] = None
```

Comment:
@snsttr Hier sollten wir nochmal weiterdiskutieren wie eine sinnvolle Abbildung aussieeht. Evtl. Abgabefrist als Zeitpunkt und Bindefrist als Zeitmenge, oder Zeitspanne?

Reply:
Wie gehen wir damit um, wenn Felder sowohl eine Zeitspanne als eine Zeitmenge beinhalten könnten, also so wie die alte COM Zeitraum?

Reply:
Mögliche Lösung: Zeitraum doch noch drinlassen, als deprectaed markeiren, aber für manche Fälle noch drinlassen, wenn es wirklcih beides sein kann. 
Alternativ: Felder duplizieren: abgabefristdauer, abgeabfristdatum

Reply:
> Wie gehen wir damit um, wenn Felder sowohl eine Zeitspanne als eine Zeitmenge beinhalten könnten, also so wie die alte COM Zeitraum?

`Optional["Zeitspanne | Zeitmenge"]`?

Reply:
Stimmt, in einer Diskussion mit Konstantin kam allerdings die Frage auf, ob das so eine gute Idee ist, da es nicht in allen Sprachen ein Pendant zum Union-type gibt. 
Da es ja an mehreren Stellen Fristen gibt, kam mir noch die Idee ein COM Frist zu erstellen, was dann beide Optionen beinhaltet. Was denkt ihr? @lord-haffi @snsttr @hf-kklein @hf-krechan @hf-aschloegl

Reply:
Naja, in den JSON-Schemas wird der Any-Of type definiert, was genau der Union-type in Python ist. Wie die einzelnen Sprachen das Pattern dann umsetzen, ist ein Problem des Code-Generators imo. Bzw. desjenigen, der das dann implementiert. Da die Typen auch disjunkt sind, sehe ich da zumindest in der Theorie auch kein Problem, da nen Union draus zu machen.
@hf-kklein Was ist deine Meinung dazu?